lib/data_models.rb

```diff
@@ -0,0 +1,133 @@
+require "ostruct"
+
+# Module DataModels provides Data Models for different objects (JSON Format) for tight coupling
+#
+# @author Piyush Wani <piyushwww13@gmail.com>
+#
+module DataModels
+  # this is a method which returns data model for acknowledgment object
+  #
+  # @param [#String] error emit event
+  # @param [#String] data data (mainly in string format)
+  # @param [#Integer] cid counter
+  # @return [#Hash] ack_object in hash format
+  #
+  def get_ack_object(error, data, cid)
+    ack_object = OpenStruct.new({
+      cid: cid,
+      data: data,
+      error: error
+    })
+    return ack_object.to_h;
```

Comment:
Please update the method comments and you can make the method more efficient.

Reply:
done

Reply:
done